### DISCUSSION TITLE
Readme.md

### DISCUSSION THREAD

**Discussion by AndyMagwayer** [521xueweihan/HelloGitHub#2591] [精选]
Hi can u share how to put github emoji into the title of your project please ![image](https://github.com/521xueweihan/HelloGitHub/assets/132371413/bdfd05e8-c6ae-4c48-b3bf-90e3090ec825)

**Comment by kieran-ryan** [comment]
Hi @AndyMagwayer! To include emojis inside your project description, you can use [GitHub markdown for emojis](https://gist.github.com/rxaviers/7360908). In this case, you will want to use `:octocat:`.

Hope this was helpful, let me know if this answers your question! 🙌

**Comment by 4pqpfn2m5g-beep** [comment]
Please search primusfilmsltd.com
